### DISCUSSION TITLE
Are Breaking changes also shown?

### DISCUSSION THREAD

**Discussion by nishant-shah-social** [Tufin/oasdiff#45] [Q&A]
Hi Team,
Looks like a promising tool for open api spec diff.
Does it also spit out whether the destination api spec has breaking changes or not?
Or it's just diff between the two?

**Comment by reuvenharrison** [comment]
There is currently no indication of whether a change is breaking or not but it should be quite easy to add.
I would start with the following:

  *   deleted endpoints, responses, params, headers
  *   modified schemas and content

From: nishant-shah-social ***@***.***>
Date: Tuesday, 21 September 2021 at 14:30
To: Tufin/oasdiff ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Tufin/oasdiff] Breaking changes? (Discussion #45)
*** CAUTION: This email was sent from an EXTERNAL source. Think before clicking links or opening attachments. ***

Hi Team,
Looks like a promising tool for open api spec diff.
Does it also spit out whether the destination api spec has breaking changes or not?
Or it's just diff between the two?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/Tufin/oasdiff/discussions/45>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAV7CFYACNRXSXFKZBPN6ODUDBUGJANCNFSM5EOFVJ7Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by reuvenharrison** [reply]
@nishant-shah-social  breaking-changes are now supported.
